src/main/java/io/quarkus/registry/app/maven/PlatformCatalogContentProvider.java

```diff
@@ -0,0 +1,132 @@
+                platformRelease.version).toString();
+        ExtensionCatalog expected = ExtensionCatalog.builder()
+                .setId(id)
+                .setBom(resolveBom(platformRelease, platform))
+                .setPlatform(true)
+                .setQuarkusCoreVersion(platformRelease.quarkusCoreVersion)
+                .setMetadata(platformRelease.metadata)
+                .setUpstreamQuarkusCoreVersion(platformRelease.upstreamQuarkusCoreVersion)
+                .setCategories(
+                        platformRelease.categories.stream().map(prc -> io.quarkus.registry.catalog.Category.builder()
+                                .setId(prc.category.categoryKey)
+                                .setName(prc.category.name)
+                                .setMetadata(prc.metadata)
+                                .setDescription(prc.category.description)
+                                .build()).toList())
+                .setExtensions(
+                        platformRelease.extensions.stream().map(pe -> Extension.builder()
+                                .setName(pe.extensionRelease.extension.name)
+                                .setDescription(pe.extensionRelease.extension.description)
+                                .setGroupId(pe.extensionRelease.extension.groupId)
+                                .setArtifactId(pe.extensionRelease.extension.artifactId)
+                                .setVersion(pe.extensionRelease.version)
+                                .setMetadata(pe.metadata)
+                                //TODO: This information is not stored in the DB
+                                .setOrigins(List.of(ExtensionOrigin.builder().setId(id).build()))
+                                .build()).toList())
+                .build();
+        return toString(expected);
+    }
+
+    /**
+     * Resolves the BOM coordinates to use for the given platform release.
+     * <p>
+     * The BOM coordinates are resolved from the {@code bom} field of the platform release if it is not {@code null}.
+     * Otherwise, the BOM coordinates are resolved from the {@code last-bom-update} metadata entry of the platform release.
+     * If the {@code last-bom-update} metadata entry is {@code null}, then the BOM coordinates are resolved from the
+     * platform release's group ID, artifact ID and version. The latter is the case for platform releases that were
+     * created before the {@code last-bom-update} metadata entry was introduced.
+     *
+     * @return the BOM coordinates to use for the given platform release
+     */
+    private static ArtifactCoords resolveBom(PlatformRelease platformRelease, Platform platform) {
+        if (platformRelease.bom != null) {
+            return ArtifactCoords.fromString(platformRelease.bom);
+        }
+        String lastBomUpdate = (String) platformRelease.metadata.get("last-bom-update");
```

Comment:
Can you be more specific about which `platform` are you referring to?

Reply:
I mean the method argument.

Reply:
Because the artifactId is different. For example, the default community platform has `artifactId` set to `quarkus-bom-quarkus-platform-descriptor` but the expected value here should be `quarkus-bom`

Reply:
Should we always use `quarkus-bom` then?

Reply:
Not possible because that information is used to locate http://localhost:8080/maven/io/quarkus/platform/quarkus-camel-bom-quarkus-platform-descriptor/2.15.3.Final/quarkus-camel-bom-quarkus-platform-descriptor-2.15.3.Final-2.15.3.Final.json for example

Reply:
> Should we always use `quarkus-bom` then?

Or do you mean to always return `quarkus-bom` regardless of the platform requested?

Reply:
Then I'd expect the member info to be passed in to the method, there shouldn't be a fallback to `quarkus-bom`.  I suppose that's what `platformRelease.bom` is. In general, it'd be less confusing if the API used proper names, otherwise it's not clear when it's about a member and when it's actually about a platform.

Reply:
I am not sure I am following.
 
This is the `bom` attribute value in the platform JSON descriptor (eg. [Camel BOM](https://repo1.maven.org/maven2/io/quarkus/platform/quarkus-camel-bom-quarkus-platform-descriptor/2.15.3.Final/quarkus-camel-bom-quarkus-platform-descriptor-2.15.3.Final-2.15.3.Final.json) or [Quarkus BOM](https://repo1.maven.org/maven2/io/quarkus/platform/quarkus-bom-quarkus-platform-descriptor/2.16.3.Final/quarkus-bom-quarkus-platform-descriptor-2.16.3.Final-2.16.3.Final.json)) which is now populated when a new platform is released. 

Because this value was not being populated previously (because we never intended to return the JSON content in the initial version), I'm falling back to what I believe is the closest to the correct value. 

Maybe I should assume that `platformRelease.bom` is not-null and provide a migration script to contain the expected values? WDYT?

Reply:
Actually I am leaning towards the latter option, as I can set the `bom` value with an SQL similar to this:  
```SQL
    SELECT
        pr.id,
        p.group_id || ':' || REPLACE(p.artifact_id, '-quarkus-platform-descriptor','') || '::pom:' || pr.version
    FROM
        platform_release pr CROSS
    JOIN
        platform_stream ps CROSS
    JOIN
        platform p
    WHERE
        pr.platform_stream_id=ps.id
        and ps.platform_id=p.id
```

Reply:
ok